<ui version="4.0" >
 <class>ContactInfoDialogClass</class>
 <widget class="QDialog" name="ContactInfoDialogClass" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>314</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <property name="topMargin" >
         <number>5</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="closeToolButton" >
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="accessibleName" >
           <string>transparentButton</string>
          </property>
          <property name="layoutDirection" >
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="popupMode" >
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel" >
        <property name="text" >
         <string>Title_DT</string>
        </property>
        <property name="margin" >
         <number>6</number>
        </property>
        <property name="indent" >
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="mainDialogFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" >
         <property name="topMargin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="topHLayout" >
           <property name="topMargin" >
            <number>9</number>
           </property>
           <item>
            <widget class="MoodBox::ContactAvatarButton" name="contactAvatarToolButton" >
             <property name="accessibleName" >
              <string>transparentButton</string>
             </property>
             <property name="iconSize" >
              <size>
               <width>74</width>
               <height>74</height>
              </size>
             </property>
             <property name="autoRaise" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" >
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" >
               <item>
                <widget class="QPushButton" name="nameLinkButton" >
                 <property name="font" >
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>8</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                   <underline>true</underline>
                   <strikeout>false</strikeout>
                  </font>
                 </property>
                 <property name="cursor" >
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="accessibleName" >
                  <string>blueButton14</string>
                 </property>
                 <property name="text" >
                  <string>UserName_DT</string>
                 </property>
                 <property name="flat" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>40</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="mottoLabel" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="frameShape" >
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="lineWidth" >
                <number>0</number>
               </property>
               <property name="text" >
                <string>Motto_DT</string>
               </property>
               <property name="wordWrap" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>236</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="infoGLayout" >
           <property name="bottomMargin" >
            <number>5</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLabel" name="genderIcoLabel" >
             <property name="minimumSize" >
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2" >
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <item>
              <widget class="QLabel" name="cityLabel" >
               <property name="text" >
                <string>City_DT</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="countryLabel" >
               <property name="text" >
                <string>Country_DT</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>0</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="1" >
            <widget class="QLabel" name="aboutLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth" >
              <number>0</number>
             </property>
             <property name="text" >
              <string>About_DT</string>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2" >
            <widget class="QLabel" name="ageLabel" >
             <property name="text" >
              <string>Age_DT</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="closeHLayout" >
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addAsFriendButton" >
          <property name="accessibleName" >
           <string>rectButton</string>
          </property>
          <property name="text" >
           <string>AddAsFriendButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton" >
          <property name="accessibleName" >
           <string>rectButton</string>
          </property>
          <property name="text" >
           <string>CloseButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>MoodBox::ContactAvatarButton</class>
   <extends>QToolButton</extends>
   <header>contactavatarbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>closeButton</tabstop>
  <tabstop>contactAvatarToolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
